File: Java/libraries/recognizers-text-date-time/src/main/java/com/microsoft/recognizers/text/datetime/utilities/TimeZoneResolutionResult.java
package com.microsoft.recognizers.text.datetime.utilities;

public class TimeZoneResolutionResult {

    private final String value;
    private final Integer utcOffsetMins;
    private final String timeZoneText;

    public TimeZoneResolutionResult(String value, Integer utcOffsetMins, String timeZoneText) {
        this.value = value;
        this.utcOffsetMins = utcOffsetMins;
        this.timeZoneText = timeZoneText;
    }

    public String getValue() {
        return this.value;
    }

    public Integer getUtcOffsetMins() {
        return this.utcOffsetMins;
    }

    public String getTimeZoneText() {
        return this.timeZoneText;
    }
}

File: .NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimeZoneResolutionResult.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

namespace Microsoft.Recognizers.Text.DateTime
{
    public class TimeZoneResolutionResult
    {
        public string Value { get; set; }

        public int UtcOffsetMins { get; set; }

        public string TimeZoneText { get; set; }
    }
}